view/theme/vier/theme.php

```diff
@@ -49,7 +49,7 @@ function vier_init(App $a)
 	if (
 		DI::mode()->has(App\Mode::MAINTENANCEDISABLED)
 		&& (
-			$args->get(0) === 'profile' && $args->get(1) === ($a->getLoggedInUserNickname() ?? '')
+			$args->get(0) === 'profile' && $args->get(1) === (DI::userSession()->getLocalUserNickname() ?? '')
```

Comment:
It looks like the `App $a` parameter of this function can be removed since it was the only place it was used.

Reply:
We still pass it, but if it’s unused it doesn’t need to be declared in the function’s signature.